src/content/basics/ha-masters/index.md

```diff
@@ -0,0 +1,169 @@
+  master node will be launched automatically. Since multiple master nodes of a cluster are running in different availability zones (AZ), this setting also protects against the risk of losing control over the cluster in the case of a single AZ downtime. Read [below](#use-of-az) for details.
+- **Load balancing of API requests.** Read-only requests to the Kubernetes API are performed
+  by all three master nodes, so that latencies stay lower overall compared to a single master
+  node which can suffer from high loads temporarily.
+
+We recommend that all productions clusters on AWS are run with high
+availability of master nodes, hence this is the default setting starting with
+release v{{% first_aws_ha_masters_version %}}.
+
+Since these benefits come at the cost of additional EC2 instances and
+additional network traffic across availability zones, it is still possible to
+create clusters with a single master node. This setting is viable for test
+purposes or clusters which don't need the high availability and resilience.
+
+## Use of availability zones {#use-of-az}
+
+When high availability master nodes is activated for a cluster, the three
+master nodes are spread over different availability zones (AZ). Depending on
+the number of AZ in the region, the logic is:
+
+- With **three or more AZs** in the region, each master node is assigned to a different AZ, selected randomly.
+- With **two AZs** in the region (as is the case in `cn-north-1`), both AZs get used. Two master nodes will share the same AZ.
+
+When [converting a single master cluster to high availability](#conversion-to-ha),
+the AZ used by the master node before is re-used. Additional AZs are assigned
+by applying the logic described above. Here, the AZ assignment of existing
+worker [node pools](/basic/nodepools/) is taken into account.
+
+## Upgrades from previous releases {#upgrades}
+
+When upgrading a cluster to release v11.4.0, the cluster will remain a single
+master cluster during and after the upgrade. The API unavailability during the
+upgrade that is typical for single master clusters will apply for this upgrade.
+
+A conversion to high availability can be triggered manually after the upgrade
+is finished.
+
+## Conversion from single master to high availability {#conversion-to-ha}
+
+Single master clusters using release v{{% first_aws_ha_masters_version %}} or
+above on AWS can be converted to master node high availability in the user
+interfaces and via the APIs.
+
+## Via the web UI {#web-ui}
+
+The web UI presents information on the master node in the cluster details page.
+Next to these details you find a button _Switch to high availability…_, unless
+the cluster is currently undergoing an upgrade. Click this button and follow
+the instructions in the web UI.
+
+## Via the CLI (`gsctl`) {#gsctl}
+
+The `gsctl` CLI provides the [gsctl update cluster](/reference/gsctl/update-cluster/) to change cluster details. Check the reference for the `--master-ha` flag.
```

Comment:
Would it make sense a version of gsctl when this flag is enabled?

Reply:
> Would it make sense a version of gsctl when this flag is enabled?

I don't understand the question.

Reply:
Sorry forget to add an `add`. I was just curious if we also release gsctl with a version and if we can specify the version here when `--master-ha` flag is available

Reply:
Done